pageserver/src/layered_repository/inmemory_layer.rs

```diff
@@ -41,8 +41,8 @@ pub struct InMemoryLayer {
     /// and protected by mutex.
     inner: Mutex<InMemoryLayerInner>,
 
-    /// Predecessor layer
-    img_layer: Option<Arc<dyn Layer>>,
```

Comment:
Had this variable name just fallen out of date?

Reply:
I separated that change to a separate commit, with commit message explaining it.